test/unit/index_spec.js

```diff
@@ -3,12 +3,12 @@ import TestUtils from 'react-addons-test-utils';
 import TiltImages from '../../app/components';
 import ReactDOM from 'react-dom';
 
-describe('TiltImages', function () {
-  Object.keys(TiltImages).forEach(function (componentName) {
-    describe(componentName, function () {
-      it('should render without error', function () {
+describe('TiltImages', () => {
+  Object.keys(TiltImages).forEach(componentName => {
+    describe(componentName, () => {
+      it('should render without error', () => {
         const Component = TiltImages[componentName];
-        const e = TestUtils.renderIntoDocument(<Component />);
+        const e = TestUtils.renderIntoDocument(<div><Component /></div>);
```

Comment:
extra div is so that we can use stateless components ... long term need to figure out what we're gonna do with defineSvg so we can make all the SVGs stateless components